wic/src/main/java/edu/asu/diging/wic/core/model/impl/ConceptText.java

```diff
@@ -0,0 +1,85 @@
+package edu.asu.diging.wic.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "ConceptText")
+public class ConceptText {
```

Comment:
Dates should be stored in ISO 8601 format and that needs to be made very clear in the javadoc. Actually, the class should probably not even have a setter for the date string but accept a OffsetDateTime object and turn that into an ISO string for storage.